src/standalone-entry.tsx

```
@@ -49,73 +49,86 @@ if (rootElement) {
       },
-      on: (event: string, callback: Function) => {
-        console.log(`[Standalone Mock] Event listener for "${event}" registered.`);
-        return () => console.log(`[Standalone Mock] Event listener for "${event}" unregistered.`);
+      on: (event: string, _callback: (...args: unknown[]) => void): void => {
+        console.log(
+          `[Standalone Mock] Event listener for "${event}" registered.`,
+        );
+        return (): void =>
+          console.log(
+            `[Standalone Mock] Event listener for "${event}" unregistered.`,
+          );
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)